arangod/Network/ConnectionPool.cpp

```diff
@@ -234,13 +234,14 @@ ConnectionPtr ConnectionPool::selectConnection(std::string const& endpoint,
                                                ConnectionPool::Bucket& bucket,
                                                bool& isFromPool) {
   using namespace std::chrono;
-  const milliseconds ttl(_config.idleConnectionMilli);
+  milliseconds const ttl(_config.idleConnectionMilli);
   
-  std::lock_guard<std::mutex> guard(bucket.mutex);
-
   auto start = steady_clock::now();
```

Comment:
Do you really want measure time with lock time?

Reply:
Yes, I think measuring the time including locking is even better, because it more correctly indicates how long things took.

Reply:
Ok